airflow/providers/google/BACKPORT_README.md

```diff
@@ -0,0 +1,26 @@
+<!--
+ Licensed to the Apache Software Foundation (ASF) under one
+ or more contributor license agreements.  See the NOTICE file
+ distributed with this work for additional information
+ regarding copyright ownership.  The ASF licenses this file
+ to you under the Apache License, Version 2.0 (the
+ "License"); you may not use this file except in compliance
+ with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing,
+ software distributed under the License is distributed on an
+ "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ KIND, either express or implied.  See the License for the
+ specific language governing permissions and limitations
+ under the License.
+ -->
+
+##  Backport Google Package limitations:
```

Comment:
Can you add tables that compare Airflow 1.10 and Airflow 2?  It's best if it was a few tables - one table that describes new operators, the other with moved operators.

Reply:
CAn you add it please? I think you have an automated way of doing it and I might be mistaken again.

Reply:
If you want, I can prepare these tables for you.

Reply:
Please!